tests/unit/test_rdataframe.py

```diff
@@ -1,7 +1,6 @@
 import unittest
 import ROOT
-from PyRDF import RDataFrame
-from PyRDF import RDataFrameException
+from PyRDF.RDataFrame import RDataFrame, RDataFrameException
```

Comment:
Yes, we'll need to check the naming...

Reply:
Naming will be addressed in another PR :smile: